physionet-django/project/templates/project/published_project.html

```diff
@@ -347,7 +347,7 @@ <h5 class="card-header">Versions</h5>
       <!-- /.sidebar -->
     </div>
     <h2 id="files">Files</h2>
-    {% if project.allow_file_downloads %}
+    {% if has_access_to_project %}
     {% if project.deprecated_files %}
```

Comment:
deprecation warning should be moved above `has_access_to_project` as we would like to inform the user why the files are not accessible. the logic of restricting access can be solely handled by `has_access_to_project`

Reply:
same story for the embargo notification just below `project.deprecated_files`

Reply:
done